client/src/components/Overview.jsx

```diff
@@ -2,27 +2,59 @@ import axios from 'axios';
 import React from "react";
 import { connect } from 'react-redux';
 //import { FaFacebookSquare, FaInstagram, FaPinterest, FaTwitter } from 'react-icons/fa' ;
-//import { BsStar, BsStarHalf, BsStarFill } from 'react-icons/bs' ;
+import { BsStar, BsStarHalf, BsStarFill } from 'react-icons/bs';
 import StyleSelector from "./StyleSelector";
 import AddToCart from './AddtoCart';
 
 
 function setDefaultStyle(styles) {
-
+  //console.log(styles);
   const result = styles.filter((style) => style['default?'] === true);
- 
   return result;
 }
 
-class Overview extends React.Component {
- 
+//Finds and returns average of reviews
+//TODO: Handle edge case for no reviews in accordance with business docs
+function reviewAverage(ratings) {
+  let total = 0;
+  let weightedSum = 0;
+  for (let i = 1; i < 6; i++) {
+    // eslint-disable-next-line radix
+    total += parseInt(ratings[i]);
```

Comment:
can just add this rule to the linter, its like a newline rule basically, I figure it helkps us now to be able to space out code and then when we finish and refactor we can squeeze it all together and make it nice